station/app/api/api_v1/endpoints/docker_trains.py

```diff
@@ -73,9 +73,10 @@ def run_docker_train(train_id: str, run_config: DockerTrainExecution = None, db:
 @router.get("/{train_id}/config", response_model=DockerTrainConfig)
 def get_config_for_train(train_id: str, db: Session = Depends(dependencies.get_db)):
     train = docker_trains.get_by_train_id(db, train_id)
-    if not train.config:
+    if not train.config_id:
```

Comment:
maybe shouldn't be an error but rather a warning that default config will be used

Reply:
Sth. like "No config given. Default config is ..."?

Reply:
You`re right it should be fine like this